From 003ff802ab1ffe0ee2f4aed158207765f3e3a7c6 Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date: Thu, 10 Mar 2022 08:34:01 -0300
Subject: [PATCH 025/142] sunrpc: Remove extra parenthesis on comparison

clang issues a 'equality comparison with extraneous parentheses
[-Werror,-Wparentheses-equality]' in this.  Instead of suppress
the warning, just follow the next comparison style.
---
 sunrpc/key_call.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sunrpc/key_call.c b/sunrpc/key_call.c
index b0b29e09a7..1072c97015 100644
--- a/sunrpc/key_call.c
+++ b/sunrpc/key_call.c
@@ -456,7 +456,7 @@ getkeyserv_handle (int vers)
       return kcp->client;
     }
 
-  if ((kcp->client == (CLIENT *) NULL))
+  if (kcp->client == (CLIENT *) NULL)
     /* Use the AF_UNIX transport */
     kcp->client = clnt_create ("/var/run/keyservsock", KEY_PROG, vers, "unix");
 
-- 
2.34.1

